app/views/docs/view.html.erb

```diff
@@ -1,2 +1,49 @@
-
-<%= raw @result %>
+<section class="cli-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto border-bottom">
```

Comment:
If you could improve rendering for different types of commands, I added Cloud Hosting > Bigsample, which should look somehow to https://gist.github.com/rt2zz/e0a1d6ab2682d2c47746950b84c0b6ee

Reply:
Added Rouge to the project for syntax highlight.